doc/liquid_conditions.md

```diff
@@ -0,0 +1,67 @@
+# Liquid conditions examples
+
+When using liquid, a lot of things can be achieved, and here are some examples
```

Comment:
You can use Liquid to configure conditions applicable across multiple policies to <insert_purpose>

Some questions to help you:
* What's the benefit of using Liquid to configure conditions across multiple policies?
* Are there any prerequisites to use these examples?
* Are there any limitations with these examples? For instance, can these examples be used in a production environment?